Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM roland/on-core
```

Comment:
Okay. I also made a similar comment on one of Zach's PRs today. I think we should avoid putting build/assets into privately controlled repositories. @heckj see any problems with creating a RackHD org for docker hub?